transmart-gwas-plugin/grails-app/controllers/com/recomdata/grails/plugin/gwas/GWASController.groovy

```diff
@@ -861,6 +861,23 @@ class GWASController {
         response.outputStream << ret?.toString()
     }
 
+    def getExperimentSecureStudyList(){
```

Comment:
This makes me sad...

Reply:
yes, I know. I'll leave it as is though.